server/sysmem/mem_zos.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build zos
+// +build zos
+
+package sysmem
+
+func Memory() int64 {
```

Comment:
Saem here.

Reply:
Hi Derek, 

zOS's memory situation is quite a rabbit hole. 
I do agree that having the full feature set working is ideal, however, I've to start somewhere.